.claude/skills/run-meshtastic-android/driver_emulator.py

```
@@ -0,0 +1,195 @@
+import sys
+import time
+import xml.etree.ElementTree as ET
+
+SERIAL = None
+PKG = "com.geeksville.mesh.fdroid.debug"
+ACTIVITY = "org.meshtastic.app.MainActivity"
+
+
+def adb(*args, binary=False):
+    cmd = ["adb"] + (["-s", SERIAL] if SERIAL else []) + list(args)
+    r = subprocess.run(cmd, capture_output=not binary, timeout=120)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)